ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-release-4.10.yaml

```diff
@@ -76,7 +76,7 @@ tests:
     cluster_profile: aws
     workflow: openshift-upgrade-aws
 - as: e2e-nightly
-  cron: 0 23 * * *
+  cron: 47 21 14,25 * *
```

Comment:
this seems very very specific, any particular reason to run it at this time and way less often than before?

Reply:
Basically, to save money.  There was a PR in the past to reduce the frequency.  That specific time/date was picked from that PR.

Reply:
See: https://github.com/openshift/release/pull/31193

Reply:
ok then!